recipes/natural_language_processing/chatbot-pydantic-ai/app/chatbot-pydantic-ai.py

```diff
@@ -0,0 +1,187 @@
+import asyncio
+import sqlite3
+from collections.abc import AsyncIterator
+from concurrent.futures.thread import ThreadPoolExecutor
+from contextlib import asynccontextmanager
+from dataclasses import dataclass
+from datetime import datetime, timezone
+from functools import partial
+from pathlib import Path
+from typing import Any, Callable, Literal
+import os
+
+import streamlit as st
+from pydantic_ai import Agent
+from pydantic_ai.models.openai import OpenAIModel
+from pydantic_ai.messages import (
+    ModelMessage,
+    ModelMessagesTypeAdapter,
+    ModelRequest,
+    ModelResponse,
+    TextPart,
+    UserPromptPart,
+)
+
+# Example from https://ai.pydantic.dev/examples/chat-app/
+
+model_service = os.getenv("MODEL_ENDPOINT",
+                          "http://localhost:8001")
+model_service = f"{model_service}/v1"
+model_service_bearer = os.getenv("MODEL_ENDPOINT_BEARER")
```

Comment:
Should be removed

Reply:
Yes because model_service is used but I don't know why model_service_bearer landed here as it is not used